docs/changes/utils/16187.api.rst

```diff
@@ -0,0 +1,2 @@
+``IERS_Auto.open()`` now always returns a table of type ``IERS_Auto`` that contains the combination of IERS-A and IERS-B data, even if automatic updating of the IERS-A file is disabled or if downloading the new file fails.
+Previously, under those conditions, it would return a table of a different type (``IERS_B``) with only IERS-B data.
```

Comment:
Putting different sentences on different lines is a good idea, but there should also be line breaks within long sentences.